Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:11
```

Comment:
Yup I tried with `clojure:openjdk-boot` earlier, the main reason is that WFL build requires pulling different repos, some of which are still private. Which means we have to set up git and git tokens for doing so. Not sure how hard it would be on Github Actions but it's defo a pain when building it locally.

Reply:
@tbl3rd I'm wondering how feasible and stable `2.ii` is, that seems more appealing to me.

Reply:
The JDK shouldn't matter. OK to use the latest.